mitodl/ocw-hugo-themes
# video player styling for course v2

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/802

#### What's this PR do?
- Updates the style of video progress bar so it covers the complete width of video player.
- Updates toggleable tabs under the video to make them uniform/same. (`Related resources` and `optional` tab's UI made same as of `transcription` tab UI)

#### How should this be manually tested?
- Checkout this branch
- Build any course on new course theme, i.e: course-v2 (more detail in this [PR](https://github.com/mitodl/ocw-hugo-themes/pull/784)) which has videos, transcript, related resources and optional tab.
- Verify that video progress bar covers the complete width of video player.
- Verify that all the 3 toggleable tabs are same in terms of UI and functionality and they work as expected.
- Repeat the above steps on different browsers and different screen sizes. 
- **Important:** Verify that things in the previous course theme are working like before and this PR has no impact on that.


#### Screenshots (if appropriate)
<img width="845" alt="image" src="https://user-images.githubusercontent.com/93309234/180428580-a9e196ca-395d-48ef-895d-8a3a45a9dd7c.png">

<img width="856" alt="image" src="https://user-images.githubusercontent.com/93309234/180428606-36022b0e-1155-4144-b1cb-0a813e3d5a40.png">

<img width="856" alt="image" src="https://user-images.githubusercontent.com/93309234/180428928-73f009a7-34ee-4e85-8264-e3614151975c.png">
